### PR TITLE
GLS-11 Implement withdrawByOwner

### PR DESCRIPTION
**Pull request by alex-forshtat-tbk**
Improve the 'withdraw' example, forbid deposit higher than minimum_stake
Note: Set minimum stake to 0.1 ether! (Was 1 wei). Also, modified the
constants accordingly.

Added two tests:
1. For SampleRecipient's withdraw method
2. For RelayHub's deposit method

Fixed grammar.

### CODE REVIEW

File: contracts/Ownable.sol [hunk 1]
```diff
@@ -0,0 +1,72 @@
+pragma solidity ^0.5.0;
```
Comment (drortirosh) [inline contracts/Ownable.sol]: don't copy in code from somewhere else

File: contracts/RelayHub.sol [hunk 2]
```diff
@@ -70,6 +70,7 @@ contract RelayHub is RelayHubApi {
      * Unused deposited can be withdrawn with `withdraw()`
      */
     function depositFor(address target) public payable {
+        require(msg.value <= minimum_stake, "deposit too big");
```
Comment (drortirosh) [inline contracts/RelayHub.sol]: remove requirement: deposit can't depend on stake

File: contracts/SampleRecipient.sol [hunk 3]
```diff
@@ -15,9 +16,10 @@ contract SampleRecipient is RelayRecipient {
         get_relay_hub().deposit.value(msg.value)();
     }
 
-    function withdraw() public {
+    function withdraw() public onlyOwner {
```
Comment (drortirosh) [inline contracts/SampleRecipient.sol]: for this `onlyOwner` we copy in contract from openZepellin. I don't like it.

File: contracts/RelayHub.sol (hunk 2 above)
Comment (shahafn) [reply contracts/RelayHub.sol]: We remove it in a separate commit

File: contracts/SampleRecipient.sol (hunk 3 above)
Comment (shahafn) [reply contracts/SampleRecipient.sol]: fixed

**Pull request merged**